lib/liquid_voting/voting.ex

```diff
@@ -61,18 +62,10 @@ defmodule LiquidVoting.Voting do
 
   """
   def list_votes(organization_id) do
-    Tracer.with_span "LV/voting" do
```

Comment:
Why is the span being removed here?

Reply:
This relates to my general q: Should we be adding traces to the related core functions. E.g. resolver `votes/3` calls 'list_votes`. I am not sure a. if `list_votes` trace gives us useful information over the resolver trace? b. if it is outside the scope of focusing on Absinthe layer?